src/kvstore/plugins/hbase/HBaseStore.cpp

```diff
@@ -266,8 +270,10 @@ std::pair<ResultCode, std::vector<Status>> HBaseStore::multiGet(
         GraphSpaceID spaceId,
         PartitionID partId,
         const std::vector<std::string>& keys,
-        std::vector<std::string>* values) {
+        std::vector<std::string>* values,
+        bool canReadFromFollower) {
```

Comment:
If this parameter is not set, there is no need to add `UNUSED`. no big deal.

Reply:
Yeah, you are right.
By the way, is hbase going to be removed?

Reply:
> Yeah, you are right.
> By the way, is hbase going to be removed?

Yes, hbase will be removed in the future. so no big deal.

Reply:
👌